Dockerfile

```diff
@@ -14,19 +15,17 @@ COPY . /src/panacea-core
 # Set working directory for the 'panacea-core' build
 WORKDIR /src/panacea-core
 
-# Install panacea-core
-RUN make clean && make build
-
 # Get 'libwasmvm.so' from wasmvm
-RUN git clone -b v0.14.0 https://github.com/CosmWasm/wasmvm.git /src/wasmvm
+ADD https://github.com/CosmWasm/wasmvm/releases/download/v0.16.2/libwasmvm_muslc.a /lib/libwasmvm_muslc.a
```

Comment:
I think it would be safer to use the wasmvm v0.14.0 (instead of v0.16.2).
The panacea-core imports wasmd v0.17.0 which uses wasmvm v0.14.0.